src/style.css

```diff
@@ -0,0 +1,59 @@
+html {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.bar {
+  margin: 20px 8px;
+}
+
+.search {
+  width: 100%;
+  height: 70px;
+  padding: 5px 62px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 25px 50%;
+  background-size: 20px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  line-height: 22px;
+  font-size: 16px;
+  font-family: "Avenir-Book", sans-serif;
+  font-weight: 300;
+}
+
+.small-class {
+  font-size: 14px;
+  height: 42px;
+  padding: 5px 33px;
+  background-position: 12px 48%;
```

Comment:
hmm, why 48% in particular? (here and later in the code)
tried to pixel-match with a reference?

Reply:
Yes, i pixel-match with the reference. I didn't change it in the new push, should I?